ConsoleApp11/ConsoleApp11/Rabbit.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ConsoleApp11
+{
+    // Model the Rabbit class knowing that:
+
+    // a rabbit can have Blue, Red or Black eyes
+    // a rabbit's fur can be white, brown, black or 
+    // it has a gender
+    // has a birth date, and based on that you should be able to see how old the rabbit is
+    // we also know that a rabbit is a mammal that moves, sleeps and eats
+
+    public enum EyeColor { Blue, Red, Black};
+    public enum FurColor { White, Brown, Black, Grey };
+    public enum Gender { Male, Female, Unknown };
+    public class Rabbit
+    {
+        private int age = -1;
+        private readonly DateTime dateOfBirth;
+
+        public Rabbit( EyeColor eyeColor, FurColor fur, Gender gender, DateTime dateOfBirth)
+        {
+            this.EyeColor = eyeColor;
+            this.FurColor = fur;
+            this.Gender = gender;
+            this.dateOfBirth = dateOfBirth;
+        }
+
+        public EyeColor EyeColor { get; set; }
+
+        public FurColor FurColor { get; set; }
+        
+        public Gender Gender { get; set; }
+        
+        public int Age
+        {
+            get
+            {
+                if (age != -1)
+                {
```

Comment:
aici testezi daca e !=-1, dar tu ai initializat fieldul cu -1. De obicei incercam sa evitam initalizarile astea cu -, pentru a nu crea confuzii si a avea inteles in cod. Cel mai bine e sa initializezi cu 0(care e si valoarea default pt int), si sa ai logica in Age, care face calculul corect. In cazul nostru varsta, poate sa insemne ani(daca asa consideri), sau poate fi  x ani si y luni.